src/routes/routes.tsx

```diff
@@ -24,26 +24,24 @@ import * as paths from './paths';
 // look into this.
 const LazyAssetMapPage = lazy(() => import('pages/AssetMapPage'));
 
-export const Routes = () => {
+export const AppRoutes = () => {
   const hasCircles = useHasCircles();
   const hasSelectedCircle = useHasSelectedCircle();
```

Comment:
one or both of these hooks return `false` initially even for users where they will eventually return `true`. so if i load e.g. /history, even though the right page will eventually load, i'll get `No routes matched location "/history"` in the console at first.

this prevents us from putting a catch-all route (`<Route path="*" element={<Navigate to={paths.getHomePath()} replace />} />`) at the end of the logged-out routes, because it will also prevent getting to logged-in routes.

the absence of a catch-all there means that if you're a user that's not in any circle yet, and you visit a route like /allocation, you don't get redirected to the "This wallet isn't associated with a circle" page. you just get a blank page and `No routes matched location "/allocation"` in the console.

Reply:
Yes this is an existing issue and although I touched it, I don't feel like I am in it enough, that I want to resolve it. When we do the org overview panel, that will connect with this